package.json

```diff
@@ -15,6 +15,7 @@
 		"pretest": "npm run lint",
 		"test": "npm run test:browser && npm run test:ssr",
 		"bump": "node ./scripts/bump.js",
+		"test:react": "node ./scripts/react-tester.js",
```

Comment:
I guess we could improve naming here, one may wonder what is the difference between `test` and `test:react`. Maybe something like `test:versions`, `test:multiple` (both doesn't sound good enough still but maybe you will be able to come up wit better naming)?